### PR TITLE
Upgrade rise-storage component

### PR DESCRIPTION
**Pull request by donnapep**
Tests were failing with Chrome 50. Need to use latest on CCI.

### CODE REVIEW

**Comment by donnapep** [comment]
@stulees @rodrigopavezi Please review. Thx.

File: circle.yml [hunk 1]
```diff
@@ -10,6 +10,12 @@ dependencies:
   cache_directories:
     - /home/ubuntu/aws
   pre:
+    # latest stable chrome
```
Comment (stulees) [inline circle.yml]: @donnapep I remember removing this from our core widgets but looks like you need it back here. Does this mean we need to add back to all our widgets?

Comment (donnapep) [inline circle.yml]: @stulees I think so. I compared the last successful build of Image Widget to the failed one and the only difference was that CCI is now using version 2.23 of ChromeDriver, whereas before it was using 2.22. And this newer version is requiring Chrome 51 or greater.

**Comment by rodrigopavezi** [comment]
@donnapep looks good 👍 cheers

File: circle.yml (hunk 1 above)
Comment (stulees) [inline circle.yml]: @donnapep Okay, thanks.

**Pull request merged**